src/components/SuggestionsList/SuggestionsList.css

```diff
@@ -0,0 +1,23 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import "src/css/variables/colors.css";
+
+.y-suggestions-list {
+  width: 280px;
+}
+
+.y-suggestions-list--with-status .y-suggestions-list--loading {
+  height: 52px;
+}
+
+.y-suggestions-list--with-status.y-suggestions-list--with-results .y-suggestions-list--loading {
+  height: 48px;
```

Comment:
Should all these pixel measurements be in rems, so that they adapt for people with non-default font size?